src/server-middleware/api/jira/service-desk.js

```diff
@@ -11,7 +11,10 @@ const jiraData = (options, req) => {
     requestTypeId: options.serviceDesk.requestTypeId,
     requestFieldValues: {
       summary: truncate(req.body.feedback, 50),
-      description: req.body.feedback
+      description: req.body.feedback,
+      'customfield_10809': req.body.pageUrl,
+      'customfield_10810': req.body.browser,
+      'customfield_10811': req.body.screensize
```

Comment:
have the field Identifiers come through the runtime config, so they can be set as environment vars per deployment.